Source: written by Michal Šebeň <mseben at gmail.com>
Upstream: yes, VBox-Dev mailing  list, Message #4368
http://vbox.innotek.de/pipermail/vbox-dev/2011-July/004368.html
Reason: Disable vboxpci.ko build when passing --disable-kmods to configure

Index: src/VBox/HostDrivers/VBoxPci/Makefile.kmk
===================================================================
--- src/VBox/HostDrivers/VBoxPci/Makefile.kmk.orig
+++ src/VBox/HostDrivers/VBoxPci/Makefile.kmk
@@ -18,6 +18,7 @@
 SUB_DEPTH = ../../../..
 include $(KBUILD_PATH)/subheader.kmk
 if1of ($(KBUILD_TARGET), linux)
+ ifdef VBOX_WITH_VBOXDRV
 
  #
  # The driver.
@@ -83,6 +84,7 @@
 	$(QUIET)$(SED) -e "s;_VERSION_;${VBOX_VERSION_STRING};g; s;_MODULE_;vboxpci;g; s;_BUILDTYPE_;${KBUILD_TYPE};g" --output $@ $<
 	$(QUIET)$(CHMOD) 0755 $@
 
+ endif # VBOX_WITH_VBOXDRV
 endif # Supported platform.
 include $(KBUILD_PATH)/subfooter.kmk
 

